yt/frontends/parthenon/data_structures.py

```diff
@@ -164,10 +164,7 @@ def __init__(
 
         self.geometry = _geom_map[self._handle["Info"].attrs["Coordinates"]]
 
-        if self.geometry is Geometry.CYLINDRICAL:
-            axis_order = ("r", "theta", "z")
-        else:
-            axis_order = None
+        axis_order = None
```

Comment:
this would break things for the old format, ya? anyway to detect the new format so you could make this change backward compatible?

Reply:
Thanks for the review! See @pgrete 's comment below. The old version was for a draft of curvilinear coordinates that never made it into the code, so this isn't actually breaking. I think it's safe to merge this without a version check. (That said parthenon does have version checking.)